ppx/ppx_js/lib_internal/ppx_js_internal.ml

```diff
@@ -127,13 +136,22 @@ let merlin_hide =
 
 module Js : sig
   val type_ :
-    ?loc:Ast_helper.loc -> string -> Parsetree.core_type list -> Parsetree.core_type
+       ?loc:Ppxlib.Ast_helper.loc
+    -> string
+    -> Ppxlib.Parsetree.core_type list
```

Comment:
`Ppxlib` is opened at the top of the file, can we simplify the path to `Parsetree.core_type` and make the diff smaller ?

Reply:
Good catch. I've gotten rid of all `Parsetree.X` in this file.